docker-compose.yml

```diff
@@ -47,13 +47,15 @@ services:
       context: ./php
       target: ${DOCKER_PHP_IMAGE}
     env_file: ${PROJECT_LOCATION}/docker/local/.env
+    environment:
+      - SSH_AUTH_SOCK=/run/host-services/ssh-auth.sock
```

Comment:
Is this env useful ? Can't we use to avoid duplication of the path in the volumes ?

Reply:
This environment variable is mandatory in order to properly configure the SSH socket. I put it the `docker-compose.yml` to avoid any misconfiguration while editing the `env_file`.

But you can't use it in `volumes` as this environment variable is already defined on your machine with another value.